packages/lib-utils/src/components/list-view/ListView.tsx

```diff
@@ -32,6 +32,25 @@ export type ListViewProps<D> = VirtualizedTableProps<D> & {
   filters?: FilterItem[];
 };
 
+export function filterDefault<D>(data: D[], filterValues: Record<string, string[]>): D[] {
+  return data.filter((item) => {
+    let isRelevant = true;
+    Object.keys(filterValues).forEach((key) => {
```

Comment:
This seems like a use case for [Array.every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) method to simplify the logic (I hope I understood it correctly).

```ts
return data.filter((item) => {
  return Object.entries(filterValues).every(([key, values]) => {
    return typeof item[key] === 'string' &&
      values.every((v) => item[key].toLowerCase().includes(v));
  });
});
```

Also `D` type parameter should be narrowed like `filterDefault<D extends Record<string, unknown>>` otherwise the function will make implicit assumptions about its structure.